Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,20 @@
 import React from 'react';
 import './Header.css';
+import HeaderContent from './HeaderContent';
+
+
+const HeaderTitle = () => {
+    return (
+      <div className="headerContainer">
+        <div className="headerTitle">
+        
+       
+        Lambda School <span class="innerTitle"> @LambdaSchool 26 jan</span>
+        <HeaderContent />
```

Comment:
HeaderContent shouldn't be inside of HeaderTitle. They should be siblings inside of HeaderContainer. The end result would be the same if you moved a few lines of code around, but the structure and naming would make more sense.